modules/rules_jni/0.3.1/MODULE.bazel

```diff
@@ -0,0 +1,20 @@
+module(
+    name = "rules_jni",
+    version = "0.3.1",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.0.3")
+bazel_dep(name = "platforms", version = "0.0.4")
+
+download_jni_headers = use_extension(
+    "@fmeum_rules_jni//bzlmod:extensions.bzl",
```

Comment:
This should just be @rules_jni?

Reply:
Thanks, should be fixed now. +1 towards tooling that compares checked-in and release archive `MODULE.bazel` :-)